packages/members-api/index.js

```diff
@@ -275,7 +275,7 @@ module.exports = function MembersApi({
             });
         }
 
-        subscription.cancel_at_period_end = cancelAtPeriodEnd;
+        subscription.cancel_at_period_end = !!(cancelAtPeriodEnd);
```

Comment:
@allouis tried to be malicious about what we pass along to stripe and sent different types of data through this parameter. To avoid unnecessary complexity with error handling and type checking went with coercion solution. Let me know if you'd rather go with proper type checking here and maybe throw the same way Stripe API does?